recipes/forestdb/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+class ForestDBConan(ConanFile):
+    name = "forestdb"
+    description = "ForestDB is a KeyValue store based on a Hierarchical B+-Tree."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ForestDB-KVStore/forestdb"
+    topics = ("forestdb", "kv store")
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_snappy": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_snappy": False,
+    }
+
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def requirements(self):
+        if self.options.with_snappy:
+            self.requires("snappy/1.1.9")
+
+    def validate(self):
```

Comment:
```suggestion
    def validate(self):
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, 11)
```
Because ForestDB requires C++11